gdal/ogr/ogrsf_frmts/flatgeobuf/packedrtree.cpp

```diff
@@ -211,10 +222,8 @@ std::vector<std::pair<uint64_t, uint64_t>> PackedRTree::generateLevelBounds(cons
     // bounds per level in reversed storage order (top-down)
     std::vector<uint64_t> levelOffsets;
     n = numNodes;
-    for (auto size : levelNumNodes) {
-        levelOffsets.push_back(n - size);
-        n -= size;
-    }
+    for (auto size : levelNumNodes)
+        levelOffsets.push_back(n -= size);
```

Comment:
can you move the n-= size; in the line before for better code clarity ?

Reply:
unless you feel very strongly about it I would prefer to keep it this way

Reply:
as you wish